lang/cs/Org.Apache.REEF.Network.Examples/GroupCommunication/BroadcastReduceDriverAndTasks/BroadcastReduceDriver.cs

```diff
@@ -19,10 +19,8 @@
 using System.Collections.Generic;
```

Comment:
Default group is for every one.  I would like sample/test code to exercise the code we changed. 

In fact, it would be much cleaner to do and test fault tolerant logic in generic code as they have exactly same REEF/GC logic as IMRU in terms of event flow.